### ISSUE TITLE
Adapter seems to break since Quilt Loader 0.18.1-beta.30

### ISSUE THREAD

**Issue by Mg138** [QuiltMC/quilt-kotlin-libraries#62]
Some copious amount of:
```
[14:26:00] [main/WARN]: Found access to quilt-loader internal class org.quiltmc.loader.impl.launch.common.QuiltLauncherBase by qkl_core - Please don't use this, instead ask us to declare a new public API that we can guarantee backwards compatibility for!
java.lang.Throwable: null
	at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.loadClassOnly(KnotClassDelegate.java:203) ~[quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:149) ~[quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.launch.knot.KnotClassLoader$Separate.loadClass(KnotClassLoader.java:358) ~[quilt-loader-0.18.1-beta.35.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at org.quiltmc.qkl.core.adapter.KotlinAdapter.create(KotlinAdapter.kt:53) ~[final-mod-qkl-core.i0:0/:?]
	at org.quiltmc.loader.impl.util.ModLanguageAdapter.create(ModLanguageAdapter.java:74) ~[quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:115) ~[quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointStorage.lambda$getEntrypointContainers$1(EntrypointStorage.java:223) ~[quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:54) ~[quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke$0(EntrypointUtils.java:36) ~[quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:62) [quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.invokeContainer(EntrypointUtils.java:49) [quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:36) [quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.QuiltLoaderImpl.invokePreLaunch(QuiltLoaderImpl.java:1019) [quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.launch.knot.Knot.init(Knot.java:174) [quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.launch.knot.Knot.launch(Knot.java:75) [quilt-loader-0.18.1-beta.35.jar:?]
	at org.quiltmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:28) [quilt-loader-0.18.1-beta.35.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71) [NewLaunch.jar:?]
```

And then mods using QKL fail to load.

**Comment by SilverAndro** [comment]
Interesting, guess we'll see if there's an easy solution here

**Comment by SilverAndro** [comment]
Can you provide the full log by chance? That warning shouldnt break all mods

**Comment by Mg138** [comment]
Was going to add on that but then we forgot -- we found that it wasn't actually the culprit for breaking kotlin mods, we updated Quilt loader and the crash was gone.

It still spams that API line though.

**Comment by SilverAndro** [comment]
Forgot to give an update, we're waiting on either loader 0.18 to be stable or for an api we need to be backported

**Comment by Mg138** [comment]
Aight, will close this for now, since it's not QKL issue.

**Comment by SilverAndro** [comment]
nono we still have to fix this haha
Its just that we cant *right now* because we're waiting on loader

**Comment by Mg138** [comment]
Ah, thought it'd be automatically fixed if the loader updates. :P

**Comment by SilverAndro** [comment]
0.18.x is stable now so ill fix this in a day or 2

**Comment by SilverAndro** [comment]
fixed in `1.1.1` releasing very shortly (#64)
